templates/web/src/client.ts.twig

```diff
@@ -359,6 +359,9 @@ class Client {
         } else {
             switch (headers['content-type']) {
                 case 'application/json':
+                    if ('queries' in params && Array.isArray(params['queries'])) {
+                        params['queries'] = params['queries'].map((query) => query.toString());
+                    }
```

Comment:
We need to do this in the GET case, routes with queries will always be GET

Reply:
Done inside flatten() function instead.